bin/test

```diff
@@ -72,9 +76,21 @@ function downloadExecutableAndRunTests() {
     getDownloadUrl(function (downloadUrl) {
         console.log('Downloading VS Code into "' + testRunFolderAbsolute + '" from: ' + downloadUrl);
 
-        var stream = remote(downloadUrl, { base: '' })
-            .pipe(vzip.src())
-            .pipe(symdest(testRunFolder));
+        var version = downloadUrl.match(/\d\.\d\.\d/)[0].split('\.');
+        var isTarGz = downloadUrl.match(/linux/) && version[0] >= 1 && version[1] >= 4;
+
+        var stream;
+        if (isTarGz) {
+            stream = request(downloadUrl)
```

Comment:
I tried to make this consistent for some time but don't know enough about how streams work to figure out what was going wrong. From what I recall, trying to use `remote` for the `isTarGz` block put the entire URL in the path so it was downloading to something like `.vscode-test/https/vscode-update.azure-websites.net/1.4.0/...`

Trying to use `request` for the `vzip` block told me that `gulp-vinyl-zip` only supports buffers?